src/java/us/kbase/templates/module_perl_impl.vm.properties

```diff
@@ -215,47 +247,17 @@ Return the module status. This is a structure including Semantic Versioning numb
 =cut
 
 sub status {
-    my($return);
+    my $return;
     #BEGIN_STATUS
-    $return = {"state" => "OK", "message" => "", "version" => $VERSION,
-               "git_url" => $GIT_URL, "git_commit_hash" => $GIT_COMMIT_HASH};
+    $return = {
+        "state"           => "OK",
+        "message"         => "",
+        "version"         => $VERSION,
+        "git_url"         => $GIT_URL,
+        "git_commit_hash" => $GIT_COMMIT_HASH
+    };
     #END_STATUS
-    return($return);
+    return ( $return );
 }
 
 1;
-#else
-#BEGIN_HEADER
-use Bio::KBase::AuthToken;
-use installed_clients::KBaseReportClient;
-use Config::IniFiles;
-#END_HEADER
```

Comment:
Some of this is duplication of what's in the sample app, and unless you run `make`, the app that gets produced by default is just a bunch of random stanzas. I've integrated this and the sample app code together so that both the base app and the sample app are valid, sensible code without needing to run `make` / invoking the SDK.

Reply:
It threw me for even more of a loop when I first generated a perl kbase app! At least now there's a sensible impl file returned after init-ing a repo.